tests/sentry/tasks/test_sentry_apps.py

```diff
@@ -10,7 +10,7 @@
 from sentry.constants import SentryAppStatus
 from sentry.models import Rule, SentryApp, SentryAppInstallation
 from sentry.receivers.sentry_apps import *  # NOQA
-from sentry.shared_integrations.exceptions import ClientError, IgnorableSentryAppError
```

Comment:
Can you straight up remove `IgnorableSentryAppError` from the codebase, or is it still used practically somewhere?

Reply:
It's still used in a retry decorator: https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/sentry_apps.py#L43-L46